pkgs/development/python-modules/hcloud/default.nix

```diff
@@ -12,11 +12,11 @@
 
 buildPythonPackage rec {
   pname = "hcloud";
-  version = "1.6.3";
+  version = "1.7.1";
```

Comment:
Also 1.8.0 is out since 3 hours: https://github.com/hetznercloud/hcloud-python/releases/tag/v1.8.0.